### ISSUE TITLE
Android TV / tvOS scaling display inconsistency

### ISSUE THREAD

**Issue by ryanvanderpol** [facebook/react-native#19690]
## Environment
Amazon Fire TV Stick (Fire OS 5.2.6.3 / Android 5.1.1 Lollipop - 1080p @ 60Hz)
Apple TV (4th gen - tvOS 11.4 - 1080p @ 60Hz)

Environment:
  OS: macOS Sierra 10.12.6
  Node: 8.1.2
  Yarn: 1.7.0
  npm: 5.6.0
  Watchman: Not Found
  Xcode: Xcode 9.2 Build version 9C40b
  Android Studio: 2.3 AI-162.4069837

Packages: (wanted => installed)
  react: 16.3.1 => 16.3.1
  react-native: 0.55.4 => 0.55.4

## Description
I've been following #16500 and finally had a chance to try it out.  I built a pretty barebones app following [these instructions](https://facebook.github.io/react-native/docs/building-for-apple-tv.html) and deployed to both an Apple TV and a Fire TV Stick.  Below you can see photos of how my app looks on both devices.  

## Steps to Reproduce
Follow the instructions in #16500 and the [React Native docs](https://facebook.github.io/react-native/docs/building-for-apple-tv.html).  Build and deploy to Apple TV and Fire TV Stick.

## Expected Behavior
Both devices, when running at the same resolution (1080p @ 60Hz) should display the same.

## Actual Behavior
On the Apple TV, everything looks perfect, but on the Fire TV Stick it looks like everything is super zoomed in and squished.

Apple TV:
![2018-06-13 09 06 36](https://user-images.githubusercontent.com/1936460/41354037-f693f39c-6eeb-11e8-8e75-6434d13dfb2c.jpg)

Android TV / Fire TV:
![2018-06-13 09 08 01](https://user-images.githubusercontent.com/1936460/41354034-f45d9d80-6eeb-11e8-90c7-3db23bbd4957.jpg)

Side Note: I followed [the instructions](https://facebook.github.io/react-native/docs/building-for-apple-tv.html) from the docs and found one thing missing that was required for Amazon's Appstore to allow me to select a Fire TV as a target device.  The following line needs to be added to your `AndroidManifest.xml`:

```
    <uses-feature android:name="android.hardware.touchscreen" android:required="false" />
```

You can find the relevant instructions in [Amazon's documentation](https://developer.amazon.com/docs/app-submission/device-filtering-and-compatibility.html#device_filtering_for_tv_apps).

**Comment by ryanvanderpol** [comment]
I've discovered, per the Fire TV docs [here](https://developer.amazon.com/docs/fire-tv/design-and-user-experience-guidelines.html#screen-size-and-resolution) and [here](https://developer.amazon.com/docs/fire-tv/display-and-layout.html) that the "Display Resolution" is 960x540.  I have confirmed this on my device by looking at `Dimensions.get('window')`.

I'm not exactly sure how the React Native core project should handle this. 

My question, however, is as an app developer, how do I go about handling this inconsistency?
